notebooks/idr0002_save.ipynb

```diff
@@ -15,6 +15,18 @@
     "We use the example pipeline [Cell/particle counting, and scoring the percentage of stained objects](http://cellprofiler.org/examples/#PercentPositive). This pipeline is for two-channel images. A plate associated to the paper has been imported into a read-write server so we can demonstrate how to save analytical results."
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# When running the notebook on M1,\n",
+    "# you might have to remove this cell after removing the comment\n",
```

Comment:
Not sure I understand the instruction. I should

1. remove the comment
2. run the cell
3. remove the cell 

?